INTITA/Modules/ProfileSettings/SettingsProfileViewController.swift

```diff
@@ -10,46 +10,60 @@ import UIKit
 class SettingsProfileViewController: UIViewController, Storyboarded {
     
     var coordinator: SettingsProfileCoordinator?
+    
     private var isProfileEditing = false
+    private lazy var headerContentView: HeaderSettingsTableViewCell = .fromNib()
+    lazy var backButton = UIButton()
 
     let arr = [
-                "firstName", "full_name", "secondName",
-                "nickname", "birthday", "email",
-                "facebook", "linkedin", "twitter",
-                "phone", "address", "education",
-                "aboutUs", "aboutMy", "avatar",
-                "skype", "country", "city",
-                "current_job", "userStatus"
+        "firstName",
+        "secondName",
+        "nickname",
+        "birthday",
+        "country",
+        "city",
+        "address",
+        "phone",
+        "aboutMy",
+        "interests",
+        "education",
+        "prev_job",
+        "current_job",
+        "aboutUs",
+        "skype",
+        "facebook",
+        "linkedin",
+        "twitter",
+        "prefer_specializations",
+        "educform",
+        "education_shift"
             ]
```

Comment:
Probably in further implementation some of these fields will include detailed info, that will be extracted into other arrays, enums. So better to such kind of information in the model, linked to ViewController through ViewModel.

Reply:
This will be improved in the next task.